apps/hashdotdev/src/pages/roadmap.page.tsx

```diff
@@ -0,0 +1,259 @@
+    icon: <FaIcon name="diagram-project" type="regular" />,
+  },
+  {
+    label: "Get Involved",
+    href: "#get-involved",
+    icon: <FaIcon name="teddy-bear" type="regular" />,
+  },
+];
+
+const Head: FunctionComponent = () => (
+  <Container sx={{ marginBottom: 10 }}>
+    <Box>
+      <Typography variant="hashHeading4" component="h2" mb={3}>
+        Where is HASH at
+      </Typography>
+      <Typography variant="hashLargeTitle" mb={5} sx={{ lineHeight: 1.1 }}>
+        Roadmap
+      </Typography>
+      <BlueStylishDivider mb={5} />
+      <Box display="flex">
+        <Box width={{ xs: 1, md: 725 }}>
+          <Typography mb={2} sx={{ lineHeight: 1.5 }}>
+            Here you’ll find the features we’re intending to build, information
+            around the order in which we’ll be addressing them, as well as
+            functional and technical specifications.
+          </Typography>
+          <Typography sx={{ lineHeight: 1.5 }}>
+            We’ve mapped these features to use-cases, so if you’re interested in
+            using HASH for a particular thing, you can follow along (or even
+            contribute!)
+          </Typography>
+        </Box>
+        <Box
+          marginLeft={16}
+          sx={{
+            marginLeft: 16,
+            display: {
+              lg: "block",
+              xs: "none",
+            },
+          }}
+        >
+          <Typography marginBottom={1}>
+            <strong>Quick-links</strong>
+          </Typography>
+          {headingLinks.map(({ label, href, icon }) => (
+            <Link
+              key={href}
+              href={href}
+              sx={{
+                "&:hover": {
+                  [`> .${typographyClasses.root}`]: {
+                    color: ({ palette }) => palette.teal[90],
+                  },
+                },
+              }}
+            >
+              <Typography
+                display="flex"
+                alignItems="center"
+                sx={{
+                  color: ({ palette }) => palette.gray[70],
+                  fontSize: 15,
+                  fontWeight: 500,
+                }}
+              >
+                <Box component="span" minWidth={22} marginRight={1.25}>
+                  {icon}
+                </Box>
+                {label}
+              </Typography>
+            </Link>
+          ))}
+        </Box>
+      </Box>
+    </Box>
+  </Container>
+);
+
+type UseCaseItem = {
+  name: string;
+  icon: ReactNode;
+};
+
+const useCaseItems: UseCaseItem[] = [
+  {
+    name: "Knowledge Management",
+    icon: <FaIcon name="sitemap" type="light" />,
+  },
+  {
+    name: "Data Management",
+    icon: <FaIcon name="binary" type="light" />,
+  },
+  {
+    name: "Business Intelligence",
+    icon: <FaIcon name="display-chart-up" type="light" />,
+  },
+  {
+    name: "Website Building",
+    icon: <FaIcon name="browser" type="light" />,
+  },
+  {
+    name: "Internal Tools & Apps",
```

Comment:
```suggestion
    name: "Internal Tools/Apps",
```

Updated here: https://www.figma.com/file/JSX4lGEb5N94q0XlvqzN0O?node-id=1858:15241&mode=design#495627083